coreos-base/oem-packet/files/cloud-config.yml

```diff
@@ -0,0 +1,41 @@
+#cloud-config
+
+coreos:
+  units:
+    - name: oem-reset-interfaces.service
```

Comment:
Well, no, this is actually a limitation of our system that we will eventually need to correct.  When the machine comes back online post-reboot the machine doesn't have network at all until this runs. SO trying to run oem-cloudinit.service  will fail because it cannot reach our metadata service.  

The problem here is that this ends up running twice. Would it be better to place this .service file directly with ebuild so that it is disconnected from cloud-init altogether?